cdc/owner/owner.go

```diff
@@ -402,17 +378,20 @@ func (o *ownerImpl) updateMetrics(state *orchestrator.GlobalReactorState) {
 }
 
 func (o *ownerImpl) clusterVersionConsistent(captures map[model.CaptureID]*model.CaptureInfo) bool {
-	myVersion := version.ReleaseVersion
+	versions := make(map[string]struct{}, len(captures))
 	for _, capture := range captures {
-		if myVersion != capture.Version {
-			if o.logLimiter.Allow() {
-				log.Warn("the capture version is different with the owner",
-					zap.Reflect("capture", capture), zap.String("ownerVer", myVersion))
-			}
-			return false
+		versions[capture.Version] = struct{}{}
+	}
+
+	ok := version.CheckTiCDCVersion(versions)
+	if !ok {
+		if o.logLimiter.Allow() {
+			log.Warn("more than two versions instance in the cdc cluster, it's not allowed",
```

Comment:
```suggestion
			log.Warn("more than two versions instance in the cdc cluster, it's not allowed",
```
I think it's ambiguous. Does it mean there are more two TiCDC instances have different versions, eg, capture1(v6.0.0), capture2(v6.1.0) and capture3(v6.2.0)? Or does it mean major versions different, eg, v6.3.0 and v8.0.0?